sourcemap-uploader/package.json

```diff
@@ -25,19 +25,15 @@
     "build": "tsup src/index.ts src/lib.ts --format cjs,esm --dts"
   },
   "devDependencies": {
-    "@types/commander": "^2.12.2",
-    "@types/glob": "^8.0.0",
-    "@types/node": "^18.11.18",
-    "@types/node-fetch": "^2.6.2",
+    "@types/node": "^18.15.5",
+    "eslint": "^8.36.0",
     "npm-run-all": "^4.1.5",
-    "tsup": "^6.5.0"
+    "tsup": "6.6.2",
```

Comment:
FYI, `tsup >= 6.6.3` breaks the build for projects with the `composite: true` config. I'm not familiar with your build system, so will leave it to you to figure that out.

Relevant GH issue: https://github.com/egoist/tsup/issues/571